modules/hosted-che/partials/proc_creating-a-workspace-from-the-user-dashboard.adoc

```diff
@@ -18,7 +18,7 @@ This section describes how to create a workspace from the user dashboard in Ecli
 
 . Once login, the redirect to the user dashboard will happen.
```

Comment:
Please reword as follows: After logging in, you will be redirected to the user dashboard.

Reply:
fixed, please use github suggestion next time, so that it would be easy to apply